addon/components/http-errors-code.ts

```diff
@@ -0,0 +1,45 @@
+import { getOwner } from '@ember/application';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Component from '@glimmer/component';
+
+interface HTTPErrorsCodeArgs {
+  httpError: '404' | '500';
+}
+
+export default class extends Component<HTTPErrorsCodeArgs> {
+  @service declare intl: any;
```

Comment:
You can use a type for `intl` now with intl-v6

Reply:
Thought about it but we should merge the ember-intl PR first

Reply:
Done ;)